RELEASING.md

```diff
@@ -4,7 +4,14 @@ To release artifacts 'dependencies-parent', 'dependencies', and 'linkage-checker
 follow the ["Developers/Releasing" steps in go/ct4j](
 https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/releasing.md?cl=head).
 
-Note: Cloud OSS BOM release procedure [boms/cloud-oss-bom/RELEASING.md](
+To distinguish the enforcer rule release and Google Cloud Libraries BOM release, the release script
+takes `enforcer` as the first argument:
```

Comment:
Can't it also take bom? 
Should we have separate release docs for each product?

Reply:
The link to the BOM release doc is 5 lines below.

Reply:
This isn't clear. This is not just the enforcer rule release. It releases several artifacts.

Reply:
That's true. Let me think better name than `enforcer`.